src/Rewrite.hs

```diff
@@ -28,29 +28,35 @@ data Args
   = Args
       { updateEnv :: Utils.UpdateEnv,
         attrPath :: Text,
-        derivationFile :: FilePath
+        derivationFile :: FilePath,
+        derivationContents :: Text
```

Comment:
Just about every rewrite rule needs to look at the contents of the file, and while they have the path and are in the IO monad and could look it up, it makes sense to just pass it.